Makefile

```diff
@@ -99,8 +97,8 @@ $(BINDIR)/deepcopy-gen: cmd/libs/go2idl/deepcopy-gen
 	$(DOCKER) glide install --strip-vendor
 	touch $@
 
-.scBuildImage: hack/Dockerfile
-	sed "s/GO_VERSION/$(GO_VERSION)/g" < hack/Dockerfile | \
+.scBuildImage: build/build-image/Dockerfile
+	sed "s/GO_VERSION/$(GO_VERSION)/g" < build/build-image/Dockerfile | \
```

Comment:
Am I correct in assuming that "build" is now the place for "build related artifacts", not "hack" ?  If so, cool.

Is "Dockerfile" the only thing in "build-image"? If so do we need it in a dir?  If you're worried about confusion we could call it "Dockerfile-BuildImage" or something like that.